active/TestInsecureHTTPVerbs.py

```diff
@@ -18,7 +18,7 @@
 
 """
 
-insecureverbs=["HEAD", "OPTIONS", "PUT", "DELETE", "TRACE", "CONNECT", "DEBUG", "MOVE", "SEARCH", "PATCH", "MKCOL", "COPY", "LOCK", "UNLOCK", "ARBIT", "XXXX", "12AB"]
+insecureverbs=["HEAD", "TRACE", "OPTIONS", "PUT", "DELETE", "CONNECT", "DEBUG", "MOVE", "SEARCH", "PATCH", "MKCOL", "COPY", "LOCK", "UNLOCK", "ARBIT", "XXXX", "12AB"]
```

Comment:
Could you add a comment explaining why the change in the order of the methods? (In case someone wonders about the order in the future.)